### PR TITLE
[full-ci] Use OcStatusIndicators for FileDetails

### PR DESCRIPTION
**Pull request by lookacat**
## Description
See https://github.com/owncloud/web/issues/5976

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [x] Unit tests updated
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/src/components/SideBar/Details/FileDetails.vue [hunk 1]
```diff
@@ -18,26 +18,11 @@
...
-          v-oc-tooltip="linkSharesLabel"
-          appearance="raw"
-          class="oc-mr-xs"
-          :aria-label="linkSharesLabel"
-          @click="expandLinksPanel"
-        >
-          <oc-icon name="link" />
-        </oc-button>
+        <oc-status-indicators
+          v-if="indicators && indicators.length > 0"
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/Details/FileDetails.vue:22]:
```suggestion
          v-if="indicators.length"
```
is sufficient. the computed prop always returns an array.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingAndUpload</strong> https://drone.owncloud.com/owncloud/web/23594/66/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIUpload-upload_feature-L94.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23596/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23596/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-3-11-05-32-12.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23596/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2022-3-11-05-32-29.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23596/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-carol-2022-3-11-05-32-35.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23596/tracing/alice-shares-file-to-brian-alice-2022-3-11-05-34-03.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23596/tracing/alice-shares-file-to-brian-brian-2022-3-11-05-34-12.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23596/tracing/alice-shares-folder-with-file-to-brian-alice-2022-3-11-05-35-04.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23596/tracing/alice-shares-folder-with-file-to-brian-brian-2022-3-11-05-35-12.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23600/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23600/tracing/alice-shares-file-to-brian-alice-2022-3-11-06-26-50.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23600/tracing/alice-shares-folder-with-file-to-brian-alice-2022-3-11-06-28-06.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23600/tracing/alice-shares-folder-with-file-to-brian-brian-2022-3-11-06-28-13.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23604/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23604/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-3-13-01-56-31.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23604/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2022-3-13-01-56-53.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23604/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-carol-2022-3-13-01-57-02.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23604/tracing/alice-shares-file-to-brian-alice-2022-3-13-01-58-40.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23604/tracing/alice-shares-file-to-brian-brian-2022-3-13-01-58-51.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23604/tracing/alice-shares-folder-with-file-to-brian-alice-2022-3-13-01-59-53.zip 


</p></details>

File: packages/web-app-files/src/components/SideBar/Details/FileDetails.vue [hunk 2]
```diff
@@ -315,7 +278,7 @@ export default {
       this.sharedParentDir = sharePathParentOrCurrent
     }
   },
-  mounted() {
+  async mounted() {
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/Details/FileDetails.vue:281]: `async` doesn't make sense if you don't `await` anything. Probably a dev leftover?

Comment (lookacat) [reply packages/web-app-files/src/components/SideBar/Details/FileDetails.vue:281]: sorry, missed the await, both are async

**Review by kulmann — APPROVED** [review]
👍

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait3</strong> https://drone.owncloud.com/owncloud/web/23620/46/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by kulmann** [comment]
`webUITrashbinFilesFolders/trashbinFilesFolders.feature:23` is the only failing test in CI and completely unrelated (= flaky). Force-merging.

**Pull request merged**